pkg/ovs/ovn-nbctl.go

```diff
@@ -2178,3 +2189,257 @@ func (c Client) SetPolicyRouteExternalIds(priority int32, match string, nameIpMa
 	}
 	return nil
 }
+
+type DHCPOptionsUUIDs struct {
+	DHCPv4OptionsUUID string
+	DHCPv6OptionsUUID string
+}
+
+func (c *Client) createDHCPOptions(ls, cidr, optionsStr string) (dhcpOptionsUuid string, err error) {
+	klog.Infof("create dhcp options ls:%s, cidr:%s, optionStr:[%s]", ls, cidr, optionsStr)
+
+	protocol := util.CheckProtocol(cidr)
+	output, err := c.ovnNbCommand("create", "dhcp_options",
+		fmt.Sprintf("cidr=%s", strings.ReplaceAll(cidr, ":", "\\:")),
+		fmt.Sprintf("options=%s", strings.ReplaceAll(optionsStr, ":", "\\:")),
+		fmt.Sprintf("external_ids=ls=%s,protocol=%s,vendor=%s", ls, protocol, util.CniTypeName))
+	if err != nil {
+		klog.Errorf("create dhcp options %s for switch %s failed: %v", cidr, ls, err)
+		return "", err
+	}
+	dhcpOptionsUuid = strings.Split(output, "\n")[0]
+
+	return dhcpOptionsUuid, nil
+}
+
+func (c *Client) updateDHCPOptions(ls, cidrBlock, gateway, dhcpV4OptionsStr string) (dhcpV4OptionsUuid string, dhcpV6OptionsUuid string, err error) {
```

Comment:
can we separate v4 and v6 into different functions, it looks messy now

Reply:
and it only takes `dhcpV4OptionsStr` as the argument,  I suppose it will also take v6 str

Reply:
hello, i have separated v4 and v6 into different functions and added the `dhcpV6Options`, we can config it in the spec now.